content/02.main-text.md

```diff
@@ -103,15 +103,16 @@ When this repository changes, Travis CI [rebuilds](https://travis-ci.org/greenel
 If successful, the output is deployed back to GitHub (to dedicated [`output`](https://github.com/greenelab/meta-review/tree/output) and [`gh-pages`](https://github.com/greenelab/meta-review/tree/gh-pages) branches).
 As a result, https://greenelab.github.io/meta-review stays up to date with the latest HTML manuscript.
 
-Manubot uses [OpenTimestamps](https://opentimestamps.org/) to timestamp the HTML and PDF outputs on the Bitcoin blockchain before deploying to GitHub.
+The idea of the "priority of discovery" is important to science, and Vale and Hyman discuss the importance of both disclosure and validation [@doi:10.7554/eLife.16931].
```

Comment:
Do this line and the `scientific precedence` phrase below give the impression that we advocate for emphasizing the precise timing of scientific discoveries?  I like the Manubot timestamps very much.  I also like the sentiment from [PLOS](http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2005203) and others that we shouldn't give all of the credit to the authors who are first to disclose or demonstrate something.

Reply:
Yes. I agree with Andy. It should be hard to backdate or falsify the history of a manuscript. However, that does not mean all credit should go to the first manuscript. I haven't looked into the reference enough to know whether this sentence inappropriately fetishizes precedence.

Reply:
I agree with the concern around highlighting the precise timing. However, being able to say roughly when something existed is valuable for many reasons. For example, if we wanted to figure out if something could be predicted computationally before a computer discovered it. Do you have any suggested wording changes? Or should I merge and let you propose a revision?

Reply:
I don't have a preferred wording right now.  Let's merge, and I'll think about it.  Adding another example like declaring a prediction in advance could help.